### PR TITLE
Implemented code to add functionality to the list page

### PR DESCRIPTION
**Pull request by ymaheshwari1**
### Related Issues

Closes #

### Short Description and Why It's Useful

Added the functionality for the list page

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/components/Picklist-item.vue [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
-  </ion-item>
-  <ion-item detail>
-    <ion-label>Picker name(s)</ion-label>
-    <ion-note slot="end">created time</ion-note>
-  </ion-item>
-  <ion-item detail>
-    <ion-label>Picker name(s)</ion-label>
-    <ion-note slot="end">created time</ion-note>
+  <ion-item v-for="picklist in pickingList" :key="picklist.picklistId" @click="viewPicklist(picklist)" detail>
+    <ion-label>{{ user.partyName }}</ion-label>
```
Comment (adityasharma7) [inline src/components/Picklist-item.vue:3]: Is this information not available in picklist?

File: src/services/PicklistService.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import api from '@/api';
+
+const getPickingList = async (): Promise<any> => {
```
Comment (adityasharma7) [inline src/services/PicklistService.ts:3]:
```suggestion
const getPicklists = async (): Promise<any> => {
```

File: src/services/PicklistService.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  return api({
+    url: 'picklists', 
+    method: 'get',
+    cache: true
+  });
+}
+
+const getPickingItemList = async (query: any): Promise<any> => {
```
Comment (adityasharma7) [inline src/services/PicklistService.ts:11]:
```suggestion
const getPicklist = async (query: any): Promise<any> => {
```

File: src/store/modules/picklist/getters.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { GetterTree } from 'vuex'
+import PicklistState from './PicklistState';
+import RootState from '@/store/RootState'
+
+const getters: GetterTree <PicklistState, RootState> = {
+  getPickingList (state) {
```
Comment (adityasharma7) [inline src/store/modules/picklist/getters.ts:6]:
```suggestion
  getPickings (state) {
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]
@ymaheshwari1 I think we should use picklists and picklist throughout

File: src/services/PicklistService.ts (hunk 2 above)
Comment (ymaheshwari1) [reply src/services/PicklistService.ts]: Changed the name to getPicklists

File: src/services/PicklistService.ts (hunk 3 above)
Comment (ymaheshwari1) [reply src/services/PicklistService.ts]: Changed the name to getPicklist.

File: src/store/modules/picklist/getters.ts (hunk 4 above)
Comment (ymaheshwari1) [reply src/store/modules/picklist/getters.ts]: Done the changes

File: src/components/Picklist-item.vue (hunk 1 above)
Comment (ymaheshwari1) [reply src/components/Picklist-item.vue:3]: No Sir, Picklist does not contain the pickers information.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**